.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies, if possible)_.
+
+### Affected node.js version(s)
+
+_Please fill in affected version(s)_
+
+### Affected platforms
```

Comment:
I think asking for `uname -a` output (or if windows something else) is more helpful. gives us architecture, endianness, os, etc.